packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -82,62 +91,62 @@ const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = (
             "& $carousel_flipperPrevious, & $carousel_flipperNext": {
                 color: "black",
                 fill: "black",
-                background: "white",
+                background: "rgba(255, 255, 255, 0.6)",
+                border: "1px solid rgba(206, 206, 206, 0.6)",
                 "& span::before": {
                     borderColor: "black",
                 },
                 "&:hover": {
+                    background: "white",
```

Comment:
should this not be foreground/background?

Reply:
No, color should not change on theme change. These are hard-coded on purpose. A dark or light slide always has the same styles, regardless of design system theme change.

Reply:
In that case can you make these constants that are referred to as they are used multiple times.

Reply:
FYI - these exist in the vNext release branch as constants.

Reply:
Updated to consts for now